<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>416</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>416</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="list">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>60</y>
      <width>310</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #232629;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="stack">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>310</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>310</width>
      <height>221</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>310</width>
      <height>221</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #232629</string>
    </property>
   </widget>
   <widget class="List_condition" name="progress_list" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>180</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="Stack_condition" name="progress_stack" native="true">
    <property name="geometry">
     <rect>
      <x>875</x>
      <y>60</y>
      <width>180</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>180</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="input_up">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>174</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>добавление сверху</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="input_down">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>добавление с низу</string>
    </property>
   </widget>
   <widget class="frame" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1080</width>
      <height>40</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>1080</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>311</width>
      <height>34</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="input_list">
     <property name="text">
      <string>добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_list">
     <property name="text">
      <string>поиск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_list">
     <property name="text">
      <string>удалить</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>311</width>
      <height>34</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="input_stack">
     <property name="text">
      <string>добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_stack">
     <property name="text">
      <string>поиск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_stack">
     <property name="text">
      <string>удалить</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="donload_json">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>180</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>загрузить с файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_json">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>220</y>
      <width>171</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>сохранить в файл</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Stack_condition</class>
   <extends>QWidget</extends>
   <header location="global">stack_condition.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>List_condition</class>
   <extends>QWidget</extends>
   <header location="global">list_condition.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>frame</class>
   <extends>QWidget</extends>
   <header location="global">frame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
